packages/amplify-category-notifications/src/__tests__/channel-APNS.test.ts

```diff
@@ -20,48 +26,32 @@ describe('channel-APNS', () => {
   const mockCertificateConfig = {};
 
   const mockPinpointClient = {
-    updateApnsChannel: jest.fn(() => {
-      return {
-        promise: () => {
-          return new Promise((resolve, reject) => {
-            resolve(mockPinpointResponseData);
-          });
-        },
-      };
-    }),
-    updateApnsSandboxChannel: jest.fn(() => {
-      return {
-        promise: () => {
-          return new Promise((resolve, reject) => {
-            resolve(mockPinpointResponseData);
-          });
-        },
-      };
-    }),
+    updateApnsChannel: jest.fn(() => ({
+      promise: () => new Promise((resolve, __reject) => {
+        resolve(mockPinpointResponseData);
+      }),
+    })),
```

Comment:
does this compile? if so never mind; haven't seen this syntax before (returning object implicitly)

Reply:
yes